pages/1.13/release-notes/1.13.0-beta1/index.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.0 (Beta)
+title: Release notes for 1.13.0 (Beta)
+menuWeight: 15
+beta: true
+excerpt: Release notes for DC/OS 1.13.0 (Beta)
+---
+DC/OS 1.13.0 (Beta) was released on April 25, 2019.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/testing/1.13.0-beta1/commit/a55cf6cd18bea6961e2cc3c957ac8d0ee47583d5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://downloads.mesosphere.io/dcos-enterprise/testing/1.13.0-beta1/commit/9d2ee649f1ddb90728757ea24d912189aaef9a52/dcos_generate_config.ee.sh"]Download DC/OS Enterprise[/button]
+
+# Release summary
+The DC/OS 1.13.0 (Beta) release is intended for testing and evaluation purposes. You should not deploy this release in a production environment or attempt to upgrade any cluster components using this version of the software. You should only use this Beta release to install new, non-production clusters.
+
+You are encouraged to provide feedback on the new features and services at [support.mesosphere.com](https://support.mesosphere.com).
+
+# New features and capabilities
+
+DC/OS 1.13.0 (Beta) includes new features and capabilities to enhance the installation and deployment experience, simplify cluster administration, increase operational productivity and efficiency, and provide additional monitoring, alerting, logging, and reporting for better visibility into cluster activity.
+
+Some highlights for this release include:
```

Comment:
I think we need to revisit this list from wording and which features we want to highlight, we will decide on those with the PMM team

Reply:
I agree that the list needs proper vetting by the team. I just attempted to summarize the key areas as best as I could tell from the feature list.